vue.config.js

```diff
@@ -1,4 +1,5 @@
 const globals = require('./globals');
+import { name as appName } from './package.json';
```

Comment:
```suggestion
const appName = require('./package.json').name;
```

@MordRot It confusing, but you can't use the `import` **syntax** outside a *Module* file.

Just remember that in the config files in the root folder, most of the time you don't have all the language features you have inside the application code.

I'm going to commit it.